lib/ProxySQL_Admin.cpp

```diff
@@ -1268,7 +1269,7 @@ void admin_session_handler(MySQL_Session *sess, ProxySQL_Admin *pa, PtrSize_t *p
 	memcpy(query,(char *)pkt->ptr+sizeof(mysql_hdr)+1,query_length-1);
 	query[query_length-1]=0;
 
-	char *query_no_space=(char *)l_alloc(query_length);	
+	char *query_no_space=(char *)l_alloc(query_length);
```

Comment:
What branch did you create this branch from? It seems that you picked up some commits that have nothing to do with your work.